Polygon.Net/Models/NewsInfo.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Polygon.Net.Models;
+
+public class NewsInfo
+{
+    [JsonProperty("id")]
+    public string Id { get; set; }
+
+    [JsonProperty("title")]
+    public string Title { get; set; }
+
+    [JsonProperty("description")]
+    public string Description { get; set; }
+
+    [JsonProperty("author")]
+    public string Author { get; set; }
+
+    [JsonProperty("published_utc")]
+    public string Published_utc { get; set; }
+
+    [JsonProperty("keywords")]
+    public List<string> Keywords { get; set; }
```

Comment:
Do we need to initialize these List properties with a default value of an empty list?